### PR TITLE
Remove dead coinjoin implementations from docs/safety.md

### PR DESCRIPTION
**Pull request by LeoSpyke**
## Change Description
Remove dead conjoin implementations from docs.

### CODE REVIEW

File: docs/safety.md [hunk 1]
```diff
@@ -400,7 +400,7 @@ The following steps are recommended to cut all links between the old clearnet
 node and the new Tor node:
 1. Close all channels on the old node and wait for them to fully close.
 1. Send all on-chain funds of the old node through a Coin Join service (like
```
Comment (ellemouton) [inline docs/safety.md:402]: while we are here, can we fix the s/Coin Join/CoinJoin typo? 🙏

File: docs/safety.md [hunk 2]
```diff
@@ -400,7 +400,7 @@ The following steps are recommended to cut all links between the old clearnet
...
-   Wasabi or Samurai/Whirlpool) until a sufficiently high anonymity set is
+   Joinmarket) until a sufficiently high anonymity set is
```
Comment (ellemouton) [inline docs/safety.md:403]: s/Joinmarket/JoinMarket

File: docs/safety.md [hunk 3]
```diff
@@ -400,7 +400,7 @@ The following steps are recommended to cut all links between the old clearnet
...
    reached.
```
Comment (ellemouton) [inline docs/safety.md:404]: nit: this now fits on previous line

**Review by ellemouton — APPROVED** [review]
Thanks 🙏  a few nits

**Review by saubyk — CHANGES_REQUESTED** [review]
Recommend edit the entire sentence to below:
`If desired, take steps to preserve the onchain privacy of the funds from the old node`

Does not make sense to call out any specific method or service here

**Comment by saubyk** [comment]
Hi @LeoSpyke can take over this pr, if you don't have availability to address the recommendations

**Comment by guggero** [comment]
Replaced by https://github.com/lightningnetwork/lnd/pull/8925.

**Pull request closed without merging**